README.md

```diff
@@ -0,0 +1,73 @@
+
+<div>
+<a href="https://www.asyncapi.com"><img src="./assets/readme-banner.svg" alt="AsyncAPI Brand Banner" width="100%"></a>
+</div>
+<br>
+<h2>Welcome to the brand ✨</h2>
+<p>Hi there! Thanks for stopping by Emoji 😄 In this project, we focus on all things related to the creative development of the AsyncAPI brand. The overarching goal for this repo is to provide a space where we can collaborate to define visuals such as logos, colors, typography, iconography, and more! All are welcome to contribute ideas and work where you feel comfortable. Before you submit your contribution, please read our <a href="https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md" target="_blank">Code of Conduct</a> and <a href="https://github.com/asyncapi/asyncapi/blob/master/CONTRIBUTING.md#contributing-to-asyncapi" target="_blank">Contribution Guidelines</a>.</p>
+<br>
+<h2>Ready to contribute? 🙌 💡 </h2>
+<p>There are plenty of ways to share your contribution with this project, and you don't have to be a designer to do so! We welcome feedback and work alike in the following areas:</p>
+
+<ul>
+<li>Design System</li>
+<li>Logo design/usage</li>
+<li>Brand identity</li>
+<li>Typography</li>
+<li>Brand guidelines</li>
+<li>Iconography</li>
+<li>Mascot ideation/illustration</li>
+<li>UI Kit</li>
+</ul>
+
+<p>Issues in this repo are open to all who would like to give feedback and/or provide visual work.</p>
+<br>
+<h2>Roadmap 🚗 💨</h2>
+<p>View the chart below to learn about the current goals for the AsyncAPI brand!</p>
+<table width="100%" border style="border: 1px solid">
+<thead>
+<th>Step</th>
+<th>Goal</th>
+</thead>
+<tbody>
+<tr>
+<td align="middle">1</td>
+<td>Refresh logos, colors, typography, & establish voice/tone</td>
+</tr>
+<tr>
+<td align="middle">2</td>
+<td>Create diverse set of iconography to complement tooling, website, & visual marketing</td>
+</tr>
+<tr>
+<td align="middle">3</td>
+<td>Brand guidelines website</td>
+</tr>
+<tr>
+<td align="middle">4</td>
+<td>Ideate and illustrate mascot to complement new brand style</td>
+</tr>
+<tr>
+<td align="middle">5</td>
+<td>Design marketing campaign for brand roll-out</td>
+</tr>
+<tr>
+<td align="middle">6</td>
+<td>Develop design system and implement across products/websites</td>
+</tr>
+<tr>
+<td align="middle">7</td>
+<td>Create UI Kit - classes and react components</td>
+</tr>
+<tr>
+<td align="middle">8</td>
+<td>New website design based on design system and UI Kit</td>
+</tr>
+<tr>
+<td align="middle">9</td>
+<td>Feedback and iteration on design system and UI Kit</td>
+</tr>
+</tbody>
+</table>
+<br>
+<h2>Join the discussion</h2>
+<p>Join the <a href="https://asyncapi.com/slack-invite" target="_blank">AsyncAPI Slack workspace</a> and the <code>#12_design</code> channel to follow all conversations relating to this repo as well as other topics of design.</p>
```

Comment:
Do you want to add a `Design` category in our GH Discussions? 
https://github.com/asyncapi/community/discussions 

<img width="1549" alt="Screen Shot 2021-12-15 at 3 30 18 PM" src="https://user-images.githubusercontent.com/19964402/146280829-72bda45b-93f9-4894-9082-5ca90696feca.png">